params/version.go

```diff
@@ -24,7 +24,7 @@ import (
 const (
 	VersionMajor = 5         // Major version component of the current release
 	VersionMinor = 0         // Minor version component of the current release
-	VersionPatch = 3         // Patch version component of the current release
+	VersionPatch = 4         // Patch version component of the current release
```

Comment:
Let's use `5.1.0`

Reply:
done in [a626974](https://github.com/scroll-tech/go-ethereum/pull/515/commits/a626974ba290a8f6f32b13429466e2bb1a668c99)